### PR TITLE
Amls 4455

### PR DESCRIPTION
**Pull request by slavvco**
Accessibility bugs fix.

Fee tables description moved to caption tag, renamed button from Continue to Continue to payment on pages preceding Ways to pay page, added missing fee guidance description header for first column of the table

## Related / Dependant PRs?

none

## How Has This Been Tested?

- unit tests
- manual tests

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [x] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [x] Requires acceptance test run.
- [x] Appropriate labels added.

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -116,7 +116,7 @@ microservice {
       show-fees = false
       enrolment-store = true
       fx-enabled = true
-      phase-2-changes = false
+      phase-2-changes = true
```
Comment (matthewharris1) [inline conf/application.conf]: Should this be false?

Comment (slavvco) [reply conf/application.conf]: OMG no

Comment (slavvco) [reply conf/application.conf]: I mean 
OMG yes

Comment (slavvco) [reply conf/application.conf]: done

**Pull request merged**